example-react-app/.env.development.local

```diff
@@ -1,4 +1,6 @@
 REACT_APP_JMIX_REST_URL=/rest/
 REACT_APP_REST_CLIENT_ID=client
 REACT_APP_REST_CLIENT_SECRET=secret
-REACT_APP_GRAPHQL_URI=/graphql
\ No newline at end of file
+REACT_APP_GRAPHQL_URI=/graphql
+REACT_APP_DISABLED_PERMS_MENUS_CHECK=true
+REACT_APP_DISABLED_PERMS_SCREENS_CHECK=true
```

Comment:
Let's have a single flag and revert the logic, e.g. `REACT_APP_ENABLE_UI_PERMISSIONS=false`